src/main/java/ome/services/scripts/ScriptRepoHelper.java

```diff
@@ -77,7 +76,7 @@
      * matching scripts in the given directory.
      */
     public final static IOFileFilter BASE_SCRIPT_FILTER = new AndFileFilter(Arrays
-            .asList(new FileFilter[] { EmptyFileFilter.NOT_EMPTY,
+            .asList(new IOFileFilter[] { EmptyFileFilter.NOT_EMPTY,
```

Comment:
The fact that all of this is in the public API is less than ideal for dropping the library.

Reply:
I assume that's a response to https://github.com/ome/omero-server/pull/135#issuecomment-976338127 rather than a review of these changes :)

Reply:
Yes. I just wanted to make the comment closer to the actual code.